<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lines</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/src/img/iс.ico</normaloff>:/src/img/iс.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>1111</width>
      <height>951</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="layGame">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>270</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="layNext3"/>
   </widget>
   <widget class="QLabel" name="lblScoreTitle">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>250</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ELEGANCE</family>
      <pointsize>32</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblScore">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ELEGANCE</family>
      <pointsize>48</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblNext3Title">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>110</y>
      <width>381</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveGame">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>630</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ELEGANCE</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnSaveGame{
color: #fff;
   border-radius: 15px;
   border: 4px solid rgb(37, 204, 255);
   background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 rgb(138, 42, 255), stop: 1 rgb(214, 90, 255));
}

QPushButton#btnSaveGame:hover{
background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                                    stop: 0 rgb(142, 29, 255), stop: 1 rgb(255, 47, 245));
border: 4px solid rgb(26, 255, 236);
}
</string>
    </property>
    <property name="text">
     <string>Save game</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInfo">
    <property name="geometry">
     <rect>
      <x>1586</x>
      <y>772</y>
      <width>291</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
